addons/disposable/functions/fnc_takeLoadedATWeapon.sqf

```diff
@@ -16,19 +16,29 @@
 #include "script_component.hpp"
 
 params ["_unit"];
-TRACE_1("params",_unit);
+TRACE_1("params", _unit);
```

Comment:
Okay, is there any documentation on how spaces work? Because the only reference [here](https://ace3mod.com/wiki/development/coding-guidelines.html) is to have 4 spaces instead of tabs.